cartographer_ros/urdf/drz_telemax_complete.urdf

```diff
@@ -0,0 +1,1653 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please do not add files with proprietary robot information to public repositories.

Reply:
:+1: Done. Was not yet finished with cleaning up.